templates/page.html

```diff
@@ -61,12 +61,12 @@ <h1 class="article-title">
             {% endif %}
 
             {% if page.date %}
-                <li>{{ macros_format_date::format_date(date=page.date, short=true, language_strings=language_strings) }}</li>
+                <li>{{ macros_format_date::format_date(date=page.date, short=false, language_strings=language_strings) }}</li>
```

Comment:
❌

Reply:
In my opinion, it would be clearer to include the year, just a recommendation. I appreciate your work.

Reply:
Users can configure the date formats as they prefer :)